src/preprocess.js

```diff
@@ -56,6 +56,45 @@ ${diagramText}
   return JSON.stringify(diagramObject)
 }
 
+// @ts-check
+/**
+ * @param {string} diagramText
+ * @param {any} context
+ * @returns {string}
+ */
+module.exports.preprocessPlantUML = function (diagramText, context) {
+  // see: http://plantuml.com/en/preprocessing
+  const RegExInclude = /^(\s*)!(include(?:sub)?)\s+(.+?)(?:!(\w+))?$/i
+  const diagramLines = diagramText.split('\n')
+  const diagramProcessed = diagramLines.map(line => line.replace(
+    RegExInclude,
+    (match, ...args) => {
+      const prefix = args[0]
+      // const include = args[1].toLowerCase()
+      const url = args[2].trim()
+      // const sub = args[3]
+      let vfs = context.vfs
+      if (typeof vfs === 'undefined' || typeof vfs.read !== 'function') {
+        vfs = require('./node-fs')
+      }

```

Comment:
I will try to come up with a solution to hide this idiom

Reply:
Still use this additionally with exists function